src/main/java/Event.java

```diff
@@ -0,0 +1,14 @@
+public class Event extends Item{
```

Comment:
Similarly, would be good to add a spacing between Item and "{", there are a few similar cases spotted below.